### ISSUE TITLE
Change the color value of some classes in the card 

### ISSUE THREAD

**Issue by ZhangChengLin** [twbs/bootstrap#38637]
### Prerequisites

- [X] I have [searched](https://github.com/twbs/bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed feature requests
- [X] I have read the [contributing guidelines](https://github.com/twbs/bootstrap/blob/main/.github/CONTRIBUTING.md)

### Proposal

Change the color value of some classes in the card to be written like the link color, that is, the calculation of RGBA+transparency, so as to realize the function of adjusting the transparency of the font in Bootstrap itself, because there is already a transparency class of the font class, such as .text- opacity-25.

However, the current .text-opacity-25 cannot adjust the transparency of .card-text, which can be achieved by changing the writing method of related classes, which does not seem to damage too much.

The known related classes are as follows:

- .card-footer
- .card-header
- .card-title
- .card-link
- .card-body
- .card-text

.card-link currently has no specific color setting, but only involves the spacing of elements. I hope you can add some useful styles or change the name, it feels a bit mismatched between the name and the actual function.

### Motivation and context

Almost all card classes are involved. Card itself is an excellent layout structural component. It should use the newly introduced link-like operation of Bootstrap, and in the current color settings of these classes, custom attributes are undefined Yes, is it an active reserved custom name?

By the way, if it works, will it be added in the next stable version? There may not be enough time.

**Comment by julien-deramond** [comment]
Thanks for suggesting a new feature @ZhangChengLin 
Haven't read in detail, just answering to the question at the end:

> By the way, if it works, will it be added in the next stable version? There may not be enough time.

The content of the next stable version (v5.3.0) is now frozen so this feature, if accepted, won't be in it.

**Comment by SaameBirthday** [comment]
Hi @julien-deramond , i have an excellent idea and I don't know where should I share it, I'm so sorry that I'm sharing this at wrong place. So my idea is to solve the issue regarding the HTML class attribute value length, where developers get themselves lost in searching and managing the classes of bootstrap. So what we can do is we can make something like single bootstrap class for each element instead of multiple classes, and the developer can target that particular class of element and can add as many class he needs to this section, so that all the sub classes are applied to the main class 
Ex: <p bootclass="my-para">Hello world</p>
For above element:
Bootstrap class file:
.my-para{
   class1;
   class2;
   class3;
   class4;
}
Accordingly class1,2,3,4and class5 are applied to the element with class my-para


In short a separate boot.class like file which contains all the class applied to each element using class name of that element

**Comment by mdo** [comment]
Right now `.text-opacity-*` classes can only modify the other `.text-*` color utilities. I don't know when or how we'll tackle that exactly yet. You could set `.text-body` and then add your `.text-opacity-*` class.
